codegen/src/main/kotlin/com/android/designcompose/codegen/Common.kt

```diff
@@ -184,7 +184,7 @@ internal fun stringTypeToCustomizationType(strType: String): CustomizationType {
         "@Composable (ComponentReplacementContext) -> Unit" ->
             CustomizationType.ComponentReplacement
         "com.android.designcompose.ListContent" -> CustomizationType.ListContent
-        "@Composable (ImageReplacementContext) -> Bitmap?" -> CustomizationType.ImageWithContext
+        "(ImageReplacementContext) -> Bitmap?" -> CustomizationType.ImageWithContext
```

Comment:
This must need a docs update, too, right?

Reply:
Interestingly we never had a section for this customization, but I'll add one.

Reply:
Done